.github/workflows/openai.yml

```diff
@@ -6,9 +6,10 @@ name: OpenAI
 on:
   pull_request:
     branches: ['main']
-    path:
+    paths:
       - 'flaml/integrations/oai/*'
       - 'test/openai/*'
```

Comment:
```suggestion
      - 'flaml/integrations/oai/**'
      - 'test/openai/**'
```

Looks like `**` is the better wildcard character for paths. [Ref1](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), [Ref2](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths).

```
*: Matches zero or more characters, but does not match the / character. For example, Octo* matches Octocat.
**: Matches zero or more of any character.
```